src/main/webapp/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <script src = "script.js"></script>
+  </head>
+  <body onload = "authenticateUser()">
+      <div id = "createProfile">
+        <form action = "/user-data" method = "POST">
+            <label for = "name">Name:</label><br>
+            <input type="text" name="name" id = "name"><br>
```

Comment:
nits: Whitespaces should be used on both sides of any binary operator. Use them consistently.

Reply:
done.